### ISSUE TITLE
Access to npm packages

### ISSUE THREAD

**Issue by garrensmith** [pouchdb/pouchdb-server#416]
Hi @daleharvey and @marten-de-vries 

Could you give @garethbowen and myself access to the following pouchdb-server npm packages so that we can publish a new pouchdb-server release:

* couchdb-eval
* couchdb-objects
* couchdb-render
* couchdb-resp-completer
* pouchdb-bulkdocs-wrapper
* pouchdb-changeslike-wrapper
* pouchdb-plugin-error
* pouchdb-req-http-query
* pouchdb-route
* pouchdb-seamless-auth - only commandoline
* pouchdb-system-db

**Comment by marten-de-vries** [comment]
Done! If I missed any or any other packages turn up, please @mention me and I'll do my best to fix the permissions asap.

```
marten@procyon:~/git/couchdb-eval$ npm owner add garrensmith couchdb-eval
+ garrensmith (couchdb-eval)
marten@procyon:~/git/couchdb-eval$ npm owner add garrensmith couchdb-objects
+ garrensmith (couchdb-objects)
marten@procyon:~/git/couchdb-eval$ npm owner add garrensmith couchdb-render
+ garrensmith (couchdb-render)
marten@procyon:~/git/couchdb-eval$ npm owner add garrensmith couchdb-resp-completer
+ garrensmith (couchdb-resp-completer)
marten@procyon:~/git/couchdb-eval$ npm owner add garrensmith pouchdb-bulkdocs-wrapper
+ garrensmith (pouchdb-bulkdocs-wrapper)
marten@procyon:~/git/couchdb-eval$ npm owner add garrensmith pouchdb-changeslike-wrapper
+ garrensmith (pouchdb-changeslike-wrapper)
marten@procyon:~/git/couchdb-eval$ npm owner add garrensmith pouchdb-plugin-error
+ garrensmith (pouchdb-plugin-error)
marten@procyon:~/git/couchdb-eval$ npm owner add garrensmith pouchdb-req-http-query
+ garrensmith (pouchdb-req-http-query)
marten@procyon:~/git/couchdb-eval$ npm owner add garrensmith pouchdb-route
+ garrensmith (pouchdb-route)
marten@procyon:~/git/couchdb-eval$ npm owner add garrensmith pouchdb-seamless-auth
+ garrensmith (pouchdb-seamless-auth)
marten@procyon:~/git/couchdb-eval$ npm owner add daleharvey pouchdb-seamless-auth
+ daleharvey (pouchdb-seamless-auth)
marten@procyon:~/git/couchdb-eval$ npm owner add garrensmith pouchdb-system-db
+ garrensmith (pouchdb-system-db)
marten@procyon:~/git/couchdb-eval$ npm owner add nolanlawson pouchdb-seamless-auth
+ nolanlawson (pouchdb-seamless-auth)
marten@procyon:~/git/couchdb-eval$ npm owner add garethbowen couchdb-eval
+ garethbowen (couchdb-eval)
marten@procyon:~/git/couchdb-eval$ npm owner add garethbowen couchdb-objects
+ garethbowen (couchdb-objects)
marten@procyon:~/git/couchdb-eval$ npm owner add garethbowen couchdb-render
+ garethbowen (couchdb-render)
marten@procyon:~/git/couchdb-eval$ npm owner add garethbowen couchdb-resp-completer
+ garethbowen (couchdb-resp-completer)
marten@procyon:~/git/couchdb-eval$ npm owner add garethbowen pouchdb-bulkdocs-wrapper
+ garethbowen (pouchdb-bulkdocs-wrapper)
marten@procyon:~/git/couchdb-eval$ npm owner add garethbowen pouchdb-changeslike-wrapper
+ garethbowen (pouchdb-changeslike-wrapper)
marten@procyon:~/git/couchdb-eval$ npm owner add garethbowen pouchdb-plugin-error
+ garethbowen (pouchdb-plugin-error)
marten@procyon:~/git/couchdb-eval$ npm owner add garethbowen pouchdb-req-http-query
+ garethbowen (pouchdb-req-http-query)
marten@procyon:~/git/couchdb-eval$ npm owner add garethbowen pouchdb-route
+ garethbowen (pouchdb-route)
marten@procyon:~/git/couchdb-eval$ npm owner add garethbowen pouchdb-seamless-auth
+ garethbowen (pouchdb-seamless-auth)
marten@procyon:~/git/couchdb-eval$ npm owner add garethbowen pouchdb-system-db
+ garethbowen (pouchdb-system-db)
```

**Comment by garrensmith** [comment]
awesome. Thanks.

**Comment by garethbowen** [comment]
I published 4.2.0 today and the permissions went smoothly. Thanks!
